src/solvers/td3.ml

```diff
@@ -627,12 +629,27 @@ module WP =
             )
         in
 
+        let force_reanalyze = StringSet.of_list @@ GobConfig.get_string_list "incremental.force-reanalyze.funs" in
+        let eager = (not (GobConfig.get_bool "incremental.reluctant.on")) in
+        let reanalyze_entry f =
+          (* destabilize the entry points of a changed function when reluctant is off,
+             or the function is to be force-reanalyzed  *)
+          eager || StringSet.mem f.svar.vname force_reanalyze
+        in
```

Comment:
I guess that could be covered by #596, but I'm a bit hesitant on doing that since it will make the master vs interactive branch TD3s significantly more different and thereby complicating merges (either way) between the branches.

But for now, a possibility might be an extra field in `change_info` about force reanalyze functions and the current logic inside the solver could look that up instead of dealing with the option. The incremental preprocessing would then be in charge of populating the list appropriately.